### ISSUE TITLE
RBM : Number of hidden neurones

### ISSUE THREAD

**Issue by FrancoisLasson** [FrancoisLasson/Temporal_DBN#4]
This issue answers #1

In order to know the influence of the hidden layer size, we train several RBMs in the same conditions using the hidden layer size as factor. We check results (cost evolution) for 3 values : 30 neurones, 50 neurones and 100 neurones.

_Condition
number of epochs = 10000, learning rate = 1e-3, batch_size=100
9 gestures, without shuffle_

Results are reported in the following plots :

![graph rbm nb hidden](https://cloud.githubusercontent.com/assets/11289891/14352083/03ed2f32-fcd3-11e5-8881-86589a4e6ae9.png)

As we could see on these plots, the number of neurones in the hidden layer don't have a really influence on the cost evolution (100 is quiet better than 30 but not significantly). Yet, after concatenation, the big hidden layers of RBMs generate huge visible layer and past visible layer of the CRBM. So, it pretty sure that the hidden sizes of RBMs impact the training of CRBM.

In order to check this influence, we have to do a second experiment : Check the impact of the hidden layer sizes on the CRBM cost and why not, on the PER.
